backend/baseAPI/tests/test_getSounds.py

```diff
@@ -48,18 +48,37 @@ def test_response_404_nonFloat_durations(self):
 
     def test_response_400_notes_outrange(self):
         """ test if 400 with outrange notes"""
-        API_URL = "http://localhost:8000/api/sounds/0/2.0/piano"  # notes should be >= 1, <= 12
+        API_URL = "http://localhost:8000/api/sounds/0/2.0/piano/0"  # notes should be >= 1, <= 12
         response = self.client.get(API_URL)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
     def test_response_400_durations_outrange(self):
         """ test if 400 with outrange durations"""
-        API_URL = "http://localhost:8000/api/sounds/3/0.5/piano"  # durations should be >= 1, <= 6
+        API_URL = "http://localhost:8000/api/sounds/3/0.5/piano/0"  # durations should be >= 1, <= 6
         response = self.client.get(API_URL)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
     def test_response_400_nonexisting_instrument(self):
         """ test if 400 with non-existing instrument """
-        API_URL = "http://localhost:8000/api/sounds/5/1.5/bass"
+        API_URL = "http://localhost:8000/api/sounds/5/1.5/bass/0"
+        response = self.client.get(API_URL)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        
+    # ADDED Familiar sounds tests      
+    def test_response_404_without_familair_params(self):
+        """ test if 404 without instrument """
+        API_URL = "http://localhost:8000/api/sounds/4/1.5/piano"
+        response = self.client.get(API_URL)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+    
+    def test_response_404_with_bad_familair_params1(self):
+        """ test if 404 without instrument """
+        API_URL = "http://localhost:8000/api/sounds/4/1.5/piano/-1" # familair should be > 0
+        response = self.client.get(API_URL)
+        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+    
```

Comment:
can a test/assert be added to make sure that it's actually returning one of the familiar tunes/a tune of a certain length?